src/components/AddItem.js

```diff
@@ -29,48 +54,82 @@ const AddItem = ({ token, results }) => {
     }
   }
 
+  function handleSetFrequency(event) {
+    setFrequency(Number(event.target.value));
+  }
+
   return (
     <form onSubmit={event => handleSubmitForm(event)}>
-      <label htmlFor="add-shopping-list-item">Item Name</label>
-      <input
-        required
-        value={name}
-        onChange={e => setName(e.target.value)}
-        id="add-shopping-list-item"
-      />
-      <fieldset>
-        <legend>How soon will you buy this again?</legend>
-        <div>
-          <input
-            onChange={e => setFrequency(parseInt(e.target.value))}
-            type="radio"
+      <Box>
+        <Box>
+          <label htmlFor="add-shopping-list-item">Item Name</label>
+        </Box>
+        <input
+          required
+          value={name}
+          onChange={e => setName(e.target.value)}
+          id="add-shopping-list-item"
+        />
+      </Box>
+      <FormControl component="fieldset">
+        <FormLabel component="legend">
+          <Typography variant="subtitle1">
+            When will you purchase this item?
+          </Typography>
+        </FormLabel>
+        <RadioGroup
+          defaultValue={604800000}
+          aria-label="purchase frequency"
+          name="purchase-frequency"
+          value={frequency}
+          onChange={handleSetFrequency}
+        >
+          <FormControlLabel
+            value={604800000}
+            control={<Radio color="default" />}
+            label="Soon"
             id="soon"
             name="frequency"
-            value={604800000}
-            checked={frequency === 604800000}
+            labelPlacement={'start'}
+            className={classes.formControlLabel}
           />
-          <label htmlFor="soon">Soon</label>
-          <input
-            onChange={e => setFrequency(parseInt(e.target.value))}
-            type="radio"
+          <Typography className={classes.caption} variant="caption">
+            Within 7 days
+          </Typography>
+          <Divider />
+          <FormControlLabel
+            value={1209600000}
+            control={<Radio color="default" />}
+            label="Kind of Soon"
             id="kind-of-soon"
             name="frequency"
-            value={1209600000}
+            labelPlacement="start"
+            className={classes.formControlLabel}
           />
-          <label htmlFor="kind-of-soon">Kind of Soon</label>
-          <input
```

Comment:
It might be nice to use one of the material-ui [text fields](https://material-ui.com/components/text-fields/) just to keep with the style/theme!